apps/easyocr/src/main.py

```
@@ -0,0 +1,194 @@
+
+  if content_length and int(content_length) > FILE_MAX_SIZE:
+    logger.error("File too large: ", f"{request.content_length} / {FILE_MAX_SIZE}")
+    return Response(content=f"File too large, maximum size is {FILE_MAX_SIZE / 1024 / 1024:.1f}MB", status_code=413)
+
+  random_uuid = uuid.uuid4()
+  extname = content_type.split('/')[1]
+  file_path = tmp_path / f"upload-{random_uuid}.{extname}"
+
+  try:
+    await stream_file(file_path, request.stream())
+  except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)